protocol/avdl/keybase1/kbfs_git.avdl

```diff
@@ -0,0 +1,14 @@
+@namespace("keybase.1")
+
+// A protocol that KBFS listens on, for services Git requests that electron or the CLI
+// initiate.
+protocol KBFSGit {
+  import idl "common.avdl";
+  import idl "favorite.avdl";
+
+  /**
+   * createRepo creates a bare empty repo on KBFS under the given name in the given TLF.
+   * It returns the ID of the repo created.
+   */
+  RepoID createRepo(Folder folder, GitRepoName name);
```

Comment:
What do you want to do in the case where the repo is not created because the repo already exists -- either with the same exact name, or as a name with different casing?  Like github, we lower-case all repo names internally to avoid problems with case-insensitive filenames, but we're preserving the original format of the name to show in the GUI.

Do you want an error in that case?  Or should we expand the return type to return two more pieces of info --  a boolean on whether or not this repo was successfully created (i.e., it didn't already exist), and if it did already exist, the user-formatted version of the repo that already existed.

Reply:
Good Q, I'd say it's an error.  We can propagate the typed error all the way back up to the UI. Thanks!